### DISCUSSION TITLE
How can  we move a user from a custodian org to a new org?

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-Lern/Community#51] [Query] [answered]
@reshmi-nair 
We want to move some custodian user to a non-custodian org because we can not publish program for a custodian org.
Can we achieve this using **User migrate API**? 

API - **https://staging.sunbirded.org//private/user/v1/migrate**
Body -
 ```
{​
  "request": {​
    "userId": "0e59b33f-d829-459f-a043-03ea726bf91c",​
    "channel": "No Data",​
    "action": "accept",​
    "feedId": "2e59b34f-d834-989z-a043-34ea846bf91d",​
    "userExtId": "randomUserId"​
  }​
​}

```
What values should we give to the feedId and userExtId, if we want to move a userId **0a544b58-50f9-4da0-9c03-dbbb0b9dd486** to any non-custodian org ?

cc : @aks30

**Comment by rhwarrier** [comment]
@reshmi-nair  please take a look.

**Comment by BharathwajShankar** [comment]
Hi @githubpiyush yes we can migrate a custodian user to a non-custodian org(tenant) using tenant migration API(https://staging.sunbirded.org//private/user/v1/migrate).Please find the correct request for the same as below,
{
    "request": {
        "userId": "0e59b33f-d829-459f-a043-03ea726bf91c",
        "channel": "No Data",
        "externalIds": [{
            "id":"stag1-1",
            "idType":"No Data",
            "provider":"No Data"
        }]
    }
}
Here the external id is nothing but the Teacher's id of the particular state org which wants to get migrated.

**Reply by BharathwajShankar** [reply, marked as answer]
idType and provider will be org id of the user
